File: challenges/javajavajavajavajavajavajava-mcb33myj/TestRunner.java
/**
 * Test Runner für JAVAJAVAJAVAJAVAJAVAJAVAJAVA
 */
public class TestRunner {
    public static void main(String[] args) {
        System.out.println("🎯 JAVAJAVAJAVAJAVAJAVAJAVAJAVA");
        System.out.println("Führe Tests aus...\n");
        
        try {
            Tests.runAllTests();
        } catch (Exception ex) {
            System.out.println("❌ Fehler: " + ex.getMessage());
            System.exit(1);
        }
        
        System.out.println("\nTests abgeschlossen.");
    }
}

File: challenges/hallloohallloohallloohallloohallloo-mcbk503a/TestRunner.cs
using System;

public class TestRunner
{
    public static void Main(string[] args)
    {
        Console.WriteLine("🎯 HalllooHalllooHalllooHalllooHallloo");
        Console.WriteLine("Führe Tests aus...\n");
        
        try
        {
            Tests.RunAllTests();
        }
        catch (Exception ex)
        {
            Console.WriteLine($"❌ Fehler: {ex.Message}");
            Environment.Exit(1);
        }
        
        Console.WriteLine("\nTests abgeschlossen.");
    }
}